README.md

```diff
@@ -311,6 +310,28 @@ Widget build(BuildContext context) {
 
 ```
 
+Instead of `BlocActionMixin` you can use one of our classes: `ActionCubit` or `ActionBloc`
+
+```dart
+class MessageActionCubit extends ActionCubit<BuildState, Action> {
```

Comment:
Please implement generic params as you would do in example code 

so I assume
`class MessageActionCubit extends ActionCubit<BuildState, String> {`

And EventCubit doesn't have the event so we can leave BlocEvent

`class MessageActionBloc extends ActionBloc<BuildState, BlocEvent, String> {`

Reply:
I think it will show better where goes which (generic) type

Reply:
right, done